kcidev/subcommands/fetch-logs.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
```

Comment:
This script is not needed as is not connected to kci-dev

Reply:
kernel maintainer can do wget on the results from the dashboard for now, in the future would be nice to add such feature to the results command.

Reply:
We can add this in a second PR

Reply:
removed